examples/README.md

```diff
@@ -0,0 +1,132 @@
+# Examples Index
+
+This folder contains examples usage for the different features
+supported by the `neo4j-graphrag` package:
+
+- Build Knowledge Graph from PDF or text
+- Retrieve information from the graph
+- Answer questions
```

Comment:
Instead of "answer", shall we have "Question Answering" / qa, as this is the common term in literature

Reply:
https://github.com/neo4j/neo4j-graphrag-python/pull/146/commits/d44df32c8a8772e252959e00c9d9eb4c12457e04